Modules/Loadable/Segmentations/Resources/UI/qSlicerSegmentationsModule.ui

```diff
@@ -119,7 +122,7 @@
        </layout>
       </item>
       <item>
-       <widget class="qMRMLSegmentsTableView" name="SegmentsTableView">
+       <widget class="qMRMLSegmentsTableView" name="SegmentsTableView" native="true">
```

Comment:
Nitpick: Could you consider excluding this type of change.

Reply:
I don't even know what it means, apparently Qt Designer added it at some point. I'll change it back, thanks!

Reply:
What about
156: <widget class="qMRMLSegmentationDisplayNodeWidget" name="SegmentationDisplayNodeWidget" native="true"/>
Remove the native attribute here as well?

Reply:
this one too

Reply:
Thanks

Reply:
At some point, we will do a "one shot" pass on all ui files to update them.